### DISCUSSION TITLE
Failed to send notification email

### DISCUSSION THREAD

**Discussion by pixelandbrick** [verbb/formie#2180] [Q&A] [answered]
When processing an email notification on form submit, I receive the following error when sending to the System Email assigned in Craft > Settings > Email:

"Failed to send notification email: {"error":"Notification email error. No recipient email address found."}."

I can successfully able to send a email to a manual address. I'm assuming this is not a bug, but I'm stumped even after some research. Any ideas, or other things to check?

**Comment by engram-design** [comment, marked as answer]
This probably needs to be filed as an issue, as I need to know what Formie/Craft version you're on. I also need to know what your email notification settings look like, particularly the Recipients setting, whether you're using a field variable in it or not. I know you mentioned the System Email, but just checking if that's using the variable as well.

**Reply by pixelandbrick** [reply]
Thank you, submitted at https://github.com/verbb/formie/issues/2183
